### PR TITLE
Scaffold Plane2 docs

### PR DESCRIPTION
**Pull request by paulgb**
Preview link: https://plane-t2lp-hn7nn0fdu-drifting-corp.vercel.app/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@
 [![Build Docker Image](https://github.com/drifting-in-space/plane/actions/workflows/plane2-build-image.yml/badge.svg)](https://github.com/drifting-in-space/plane/actions/workflows/plane2-build-image.yml)
 [![Tests](https://github.com/drifting-in-space/plane/actions/workflows/plane2-tests.yml/badge.svg)](https://github.com/drifting-in-space/plane/actions/workflows/plane2-tests.yml)
 
-Plane is infrastructure software for **running stateful WebSocket backends at scale**. Plane is heavily inspired by [Figma’s mulitplayer infrastructure](https://www.figma.com/blog/rust-in-production-at-figma/), which dynamically spawns a process for each active document.
+Plane is a distributed system for **running stateful WebSocket backends at scale**. Plane is heavily inspired by [Figma’s mulitplayer infrastructure](https://www.figma.com/blog/rust-in-production-at-figma/), which dynamically spawns a process for each active document.
```
Comment (feliciachang) [inline README.md:16]: what does it mean for plane to be a distributed system - is there a specific type of system it is?

File: README.md [hunk 2]
```diff
@@ -13,7 +13,7 @@
...
 
 Use cases include:
```
Comment (feliciachang) [inline README.md:18]: i think we can expand on this section to include a list of features and then use cases ie https://docs.vllm.ai/en/latest/#

having a features section would help cement how plane as websocket backend is unique. some features that could be useful might explain how the backend is scaled

File: docs/pages/concepts/backend-lifecycle.mdx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+The statuses are:
+
+- `Scheduled`: The Plane controller has chosen a drone to run the backend.
+- `Loading`: The drone has acknowledged the backend and is loading the appropriate image from the registry.
+- `Starting`: The drone has loaded the image and is starting the container.
+- `Waiting`: The container has started. The drone is waiting for it to listen on an HTTP port.
+- `Ready`: The container is listening on an HTTP port. The drone is ready to route traffic to it.
+- `Terminating`: The drone has sent a request to terminate the backend. If the request was a “soft” request,
+  the backend may remain in this state for a grace period (by default 10 seoconds) before being hard-terminated.
+- `Terminated`: The drone has terminated the backend. This is considered the only terminal state.
```
Comment (rolyatmax) [inline docs/pages/concepts/backend-lifecycle.mdx:15]: What about `Swept`, `Exited`, `Failed`, and other error states?

File: README.md (hunk 1 above)
Comment (paulgb) [reply README.md:16]: Here's a good definition:

> A distributed system is a system whose components are located on different networked computers, which communicate and coordinate their actions by passing messages to one another from any system. The components interact with one another in order to achieve a common goal.

File: docs/pages/concepts/backend-lifecycle.mdx (hunk 3 above)
Comment (paulgb) [reply docs/pages/concepts/backend-lifecycle.mdx:15]: In Plane2 these are all wrapped into `Terminated`, but other data is stored (like exit code) that can be used to determine the nature of an exit. I am open to exposing that through the API, although we don't yet.

File: README.md (hunk 2 above)
Comment (paulgb) [reply README.md:18]: I'll punt this to a future PR, but if you have specific suggestions on what to add feel free to open another PR when this is merged.

**Pull request merged**